packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,81 @@
+import { BigNumberish, ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  wrapperClasses?: string;
+};
+
+/**
+ Display balance of an address
+*/
+
+export default function Balance({ address, price, wrapperClasses }: BalanceProps) {
+  const [isEthBalance, setIsEthBalance] = useState(true);
+  const [balance, setBalance] = useState<any>();
+
+  const {
+    data: fetchBalanceData,
+    isError,
+    isLoading,
+  } = useBalance({
+    addressOrName: address,
+    watch: true,
+    cacheTime: 5_000,
+  });
+
+  const onToggleBalance = () => {
+    setIsEthBalance(!isEthBalance);
```

Comment:
Since we are using this pattern (to avoid hydration) lwe can just save the Number here